Sources/Sentry/SentryCrashWrapper.m

```diff
@@ -212,6 +213,31 @@ - (void)enrichScope:(SentryScope *)scope
     [appData setValue:systemInfo[@"buildType"] forKey:@"build_type"];
 
     [scope setContextValue:appData forKey:@"app"];
+
+    // Runtime
+    NSMutableDictionary *runtimeContext = [NSMutableDictionary new];
+
+    // We set this info on the runtime context because the app context has no existing fields
+    // suitable for representing Catalyst or iOS-on-Mac execution modes. We also wanted to avoid
+    // adding two new Apple-specific fields to the app context. Coming up with a generic,
+    // reusable property on the app context proved difficult, so instead we reuse the "name"
```

Comment:
`l`: We could also add a machine-readable version such as `ios-on-macos` and `mac-catalyst` in addition to the human-readable version. Not sure if there's additional value in doing so though.